### DISCUSSION TITLE
Mirroring the .stl

### DISCUSSION THREAD

**Discussion by AndersDoverud** [supermerill/SuperSlicer#2615] [Q&A] [answered]
Where do I mirror the .stl in SuperSlicer, can't find it anywhere..?

**Comment by supermerill** [comment, marked as answer]
right click on the object -> mirror -> choose the axis.
Note: only works before rotation

**Reply by AndersDoverud** [reply]
"Note: only works before rotation"
Almost always the first thing I do, is to rotate it.
Thank you!
